pkg/utils/utils.go

```diff
@@ -41,3 +45,21 @@ func SetStringValue(values map[string]interface{}, key string, value *string) {
 		values[key] = *value
 	}
 }
+
+// SimpleMatch returns whether the given pattern matches the given text.
+// It also returns a score indicating the match between `pattern` and `text`. The higher the score the higher the match.
+// Only simple wildcard patterns are supposed to be passed, e.g. '*', 'tex*'.
+func SimpleMatch(pattern, text string) (bool, int) {
+	const wildcard = "*"
+	if pattern == wildcard {
+		return true, 0
+	}
+	if pattern == text {
+		return true, len(text)
+	}
+	if strings.HasSuffix(pattern, wildcard) && strings.HasPrefix(text, pattern[:len(pattern)-1]) {
```

Comment:
Hm, this will only work  for `foo-*`, right? What about `*-bar`?

Reply:
The latter one is not supported at the moment. Do you see a necessity of doing this as well?

Reply:
Hm, if it's easy to add then I'd add it right away now. As user I would expect that I can also use `*-bar` expression if `*` and `*-foo` is already supported.